index.html

```diff
@@ -67,7 +67,12 @@ <h3>Contiene aspirina y fármacos que ayudan a controlar la presión y el colest
 			</article>
 		</section>
 	</main>
-	<aside></aside>
+	<aside>
```

Comment:
Restore the entire file please. It has unimportant changes.

Reply:
Its just a lorem.